apps/hexagon_matmul/Makefile

```diff
@@ -0,0 +1,55 @@
+include ../support/Makefile.inc
```

Comment:
This makefile is literally a copy/paste of another app, I only changed a few command line args to the process binaries for running the app.

I totally agree that the apps need major refactoring, to use generators, etc... but I'd like to hold off on that until we can invest time in doing it consistently for all apps. What do you think?